prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -1,154 +1,183 @@
 package gov.cdc.prime.router
 
```

Comment:
@mauricereeves , it looks like you did some nice cleanup on FakeReport.kt - could you put in a brief comment explaining the changes?
Thanks.

Reply:
Yes, will do.